client/src/components/ElementEditor/HoverBar.js

```diff
@@ -97,7 +97,7 @@ class HoverBar extends Component {
       'element-editor__add-block-hover-bar',
       {
         'element-editor__add-block-hover-bar--delay': !calledInstantaneously,
-        'element-editor__add-block-hover-bar--inst': calledInstantaneously,
+        'element-editor__add-block-hover-bar--inst': calledInstantaneously
```

Comment:
This change isn't required - it's arguably nicer to leave trailing commas if they're supported - smaller diffs means less chances to conflict!

Reply:
It'd be great if usage was consistent throughout the system too, and this change is trying to bring things in line with existing code (particularly where e.g. my linter formatter goes and makes edits that are apparently contrary to someone else's...). Is there any guidance on this I'm particularly unaware of?

Reply:
Linters that automatically change existing code are very dangerous!! I've wasted a lot of time with peer reviews that have boiled down to automated changes. We should keep this kind of change to separate PRs (as mentioned above), where we can say "OK, it's automated and all LGTM so merge"